src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -222,4 +222,60 @@ public Response sendDaaRequestMessage(
       return createExceptionResponse(e);
     }
   }
+
+  @POST
+  @Consumes(MediaType.APPLICATION_JSON)
+  @RolesAllowed({ADMIN, CHAIRPERSON})
```

Comment:
Should this be Admin & Signing Official?

Reply:
I missed that, nice catch!